src/app/regular-giving/regular-giving.component.ts

```diff
@@ -170,9 +170,9 @@ export class RegularGivingComponent implements OnInit {
     next: async (mandate: Mandate) => {
       await this.router.navigateByUrl(`${myRegularGivingPath}/${mandate.id}`);
     },
-      error: (error: Error) => {
+      error: (error: {error: {error: {description?: string} }}) => {
       console.error(error);
-      const message = error.message
+      const message = error.error.error.description ?? 'Sorry, something went wrong';
```

Comment:
Probably worth refactoring soon to deal with this repeated triple error - I might do tomorrow. Maybe we can simplify the structure sent from matchbot, if not we can just dedupe some of the code to deal with it here.

![image](https://github.com/user-attachments/assets/f410d07d-1d3e-4375-8755-56a5e78e111b)